ARImageTrackingExamples/ARImageTrackingExamples/Helpers/ARHelper.swift

```diff
@@ -13,3 +13,13 @@ extension ARImageAnchor {
         return self.referenceImage.physicalSize
     }
 }
+
+extension SCNNode {
+    func opacitySlightlyTransparent() {
+        self.opacity = self.opacity * 0.95
```

Comment:
not what I meant, I was thinking you could do

```swift
extension CGFloat {
static let slightyTransparent = 0.95
static let smallOffset = 0.01
}

node.opacity = .slightlyTransparent
```

Reply:
you want to ensure the _magic numbers_ have context -- using methods ties it quite heavily to that number (for example, if you have N different opacities you want to use, you'd need N different methods)